### ISSUE TITLE
Blustream MX44VW

### ISSUE THREAD

**Issue by Miguel18-89** [bitfocus/companion-module-requests#605]
Hi Guys,

Can you develop a instance for HDMI Seamless Matrix of Blustream MX44VW?

best regards,

Miguel Cordeiro

**Comment by josephdadams** [comment]
Can you provide documentation for your device like the instructions requested?

**Comment by Miguel18-89** [comment]
I Joseph,

Many thanks for your reply.

Documentation in attach.

Best Regards,

Miguel Cordeiro

A segunda, 11/10/2021, 00:27, Joseph Adams ***@***.***>
escreveu:

**Comment by josephdadams** [comment]
No, you need to come to Github and post it manually and not via email
